oamapps/columnstoreSupport/configReport.sh

```diff
@@ -70,22 +66,22 @@ echo "-- fstab Configuration --"
 echo " "
 echo "################# cat /etc/fstab #################"
 echo " "
-$SUDO cat /etc/fstab 2>/dev/null
+cat /etc/fstab 2>/dev/null
 
 echo " "
 echo "-- Server Processes --"
 echo " "
 echo "################# ps axu #################"
 echo " "
-$SUDO ps axu
```

Comment:
also odd.  ps doesn't need root access, neither does cat, not sure why they were run with sudo.